src/app/page.js

```
@@ -140,12 +140,52 @@ export default function Home() {
+    } else if (navigator.userAgent.indexOf("Chrome") != -1) {
+      browser = 'Chrome';
+    } else if (navigator.userAgent.indexOf("Safari") != -1) {
+      browser = 'Safari';
+    } else if (navigator.userAgent.indexOf("Firefox") != -1) {
+      browser = 'Firefox';
+    } else if ((navigator.userAgent.indexOf("MSIE") != -1) || (!!document.documentMode == true)) {
+      browser = 'IE';
+    }
+    const language = navigator.language || navigator.languages[0];
+    let os = 'unknown';
+    if (navigator.userAgentData && navigator.userAgentData.platform) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)